Assignment6.Rmd

```diff
@@ -21,24 +21,30 @@ library(rpart)
 #Data
 ```{r}
 #Upload the data sets MOOC1.csv and MOOC2.csv
+
+library(dplyr)
+
 M1 <- read.csv("MOOC1.csv", header = TRUE)
 
-M2 <- 
+M2 <- read.csv("MOOC2.csv", header = TRUE)
 
 ```
 
 #Decision tree
 ```{r}
 #Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
 
-c.tree1 <- 
+c.tree1 <- rpart(certified ~ grade + assignment, method="class", data=M1) 
 
-#Check the results from the classifcation tree using the printcp() command
+#I can see assignment and grade variables in the tree. 
 
```

Comment:
If you included an unscaled version of `forum.posts` this will swamp the other two variables as the number of increments in that variable is so great.

Reply:
I see, thanks for your answer!